sepolia/config.yaml

```diff
@@ -0,0 +1,76 @@
+# Extends the mainnet preset
+PRESET_BASE: 'mainnet'
+CONFIG_NAME: 'sepolia'
+
+# Genesis
+# ---------------------------------------------------------------
+MIN_GENESIS_ACTIVE_VALIDATOR_COUNT: 1300
+# Sunday, June 19, 2022 2:00:00 PM +UTC
+MIN_GENESIS_TIME: 1655647200
+GENESIS_FORK_VERSION: 0x90000069
+GENESIS_DELAY: 86400
+
+
+# Forking
+# ---------------------------------------------------------------
+# Some forks are disabled for now:
+#  - These may be re-assigned to another fork-version later
+#  - Temporarily set to max uint64 value: 2**64 - 1
+
+# Altair
+ALTAIR_FORK_VERSION: 0x90000070
+ALTAIR_FORK_EPOCH: 50
+
+# Merge
+BELLATRIX_FORK_VERSION: 0x90000071
+BELLATRIX_FORK_EPOCH: 100
+TERMINAL_TOTAL_DIFFICULTY: 100000000000000000000000
+TERMINAL_BLOCK_HASH: 0x0000000000000000000000000000000000000000000000000000000000000000
+TERMINAL_BLOCK_HASH_ACTIVATION_EPOCH: 18446744073709551615
+
+# Capella
+CAPELLA_FORK_VERSION: 0x03001020
+CAPELLA_FORK_EPOCH: 18446744073709551615
+
+# Sharding
+SHARDING_FORK_VERSION: 0x04001020
+SHARDING_FORK_EPOCH: 18446744073709551615
+
+# Time parameters
+# ---------------------------------------------------------------
+# 12 seconds
+SECONDS_PER_SLOT: 12
+# 14 (estimate from Eth1 mainnet)
+SECONDS_PER_ETH1_BLOCK: 14
```

Comment:
hopefully this is not an issue now that we addressed hash rate volatility on Ropsten, but if we don't expect regular block times on sepolia we may want to consider this number